rules/windows/discovery_net_command_system_account.toml

```diff
@@ -11,33 +11,34 @@ adversary has achieved privilege escalation.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Net command via SYSTEM account"
 risk_score = 21
 rule_id = "2856446a-34e6-435b-9fb5-f8f040bfa7ed"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  (process.name:(whoami.exe or net.exe) or process.name:net1.exe and not process.parent.name:net.exe) and
-  user.name:SYSTEM
+process where event.type in ("start", "process_started") and 
+  user.id in ("S-1-5-18", "S-1-5-19", "S-1-5-20") and
+  process.name : ("whoami.exe", "net.exe") or
```

Comment:
```suggestion
  process.name : "whoami.exe" or
```
was going to deprecate this one, but most noise came from `net.exe`. It could still be refactored into a sequence later on.

Reply:
yeah, i say let's continue with deprecation in the other PRs and leave this as-is with @brokensound77's changes

Reply:
I removed it from deprecation after just taking out `net.exe`. Just a minor change but I'm cool either way.